src/test_creation/modules/code_analyzer/repo.py

```diff
@@ -44,7 +45,19 @@ def _get_language_file_map(self):
             for k, v in self.fileext_language_map.items():
                 if file.endswith(k):
                     file_language_map[v].append(file)
-        return file_language_map
+        return file_language_map # FIXME: why is it called file_language_map instead of language_file_map?
```

Comment:
Hi @SoloSynth1, is there a reason of calling this variable `file_language_map` instead of `language_file_map`?

Reply:
No particular reason - this code was written back in hackathon so everything was made without clear design. As you can see the function name suggests the other way (`language_file_map`), I am guessing that was an unintended oversight.

Reply:
fixed in latest commit 08b7d13